features/lost-password-page.feature

```diff
@@ -0,0 +1,35 @@
+@issue-876
+Feature:
+  As a gigadb user
+  I want to click lost password link
+  So that there is no mail list or terms and condition tick boxes
+
```

Comment:
Hi @kencho51 , this is not a clear user story as it doesn't explain the reason why a user would go on this page. Try this instead:
>As a website user
I want to reset my password
So that I can sign in to GigaDb website again

The absence of check box (use check-box, their proper name, not tick box), is part of acceptance criteria.

Reply:
Hi @rija , the test has been updated.